config/dev.webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const { resolve } = require('path');
+const config = require('@redhat-cloud-services/frontend-components-config');
+const commonPlugins = require('./plugins');
+
+const environment = process.env.ENVIRONMENT || 'stage';
+const betaOrStable = process.env.BETA ? 'beta' : 'stable';
+// for accessing prod-beta change this to 'prod-beta'
+const env = `${environment}-${betaOrStable}`;
+
+const webpackProxy = {
+  deployment: process.env.BETA ? 'beta/api/plugins' : 'api/plugins',
+  useProxy: true,
+  env,
+  appUrl: process.env.BETA ? '/beta/hac/infra' : '/hac/infra',
+  standalone: Boolean(process.env.STANDALONE),
+  ...(process.env.INSIGHTS_CHROME && {
```

Comment:
Should we also document this variable in the Readme ?

Reply:
Added